src/js/strategies/CitrixVDIStrategy.js

```diff
@@ -99,18 +97,20 @@ export default class CitrixVDIStrategy extends CCPInitiationStrategyInterface {
         return new window.CitrixWebRTC.CitrixPeerConnection(configuration, optionalConfiguration);
     }
 
-    connect(self) {
-        self._pc.onaddstream = hitch(self, self._ontrack);
-    }
-
     _ontrack(self, evt) {
-        const remoteStream = evt.stream.clone();
-
-        const audioTracks = evt.stream.getAudioTracks();
-        if (audioTracks !== undefined && audioTracks.length > 0) {
-            self._remoteAudioStream = remoteStream;
-            self._remoteAudioElement.srcObject = remoteStream;
+        window.CitrixWebRTC.mapAudioElement(self._remoteAudioElement);
+        if (evt.streams.length > 1) {
+            self._logger.warn('Found more than 1 streams for ' + evt.track.kind + ' track ' + evt.track.id + ' : ' +
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Media_Capture_and_Streams_API.  Concepts of MediaStream and track are a little jumbled here. Based on offline discussion - This has been copied from existing strategy and that piece of code has been there for a long time. In order to make major changes, we want to keep this similar.

We can take a backlog AI to cleanup this code through minimal change iterations in the future.